configs/ IFLIGHT_BORG_F435/config.h

```diff
@@ -0,0 +1,143 @@
+
+// SPI
+#define SPI1_SCK_PIN                    PA5
+#define SPI1_SDI_PIN                    PA6
+#define SPI1_SDO_PIN                    PA7
+#define SPI1_NSS_PIN                    PA4
+
+#define SPI2_SCK_PIN                    PB13
+#define SPI2_SDI_PIN                    PB14
+#define SPI2_SDO_PIN                    PB15
+#define SPI2_NSS_PIN                    PB12
+
+#define SPI3_SCK_PIN                    PB3
+#define SPI3_SDI_PIN                    PB4
+#define SPI3_SDO_PIN                    PB5
+#define SPI3_NSS_PIN                    PA15
+
+// GYRO & ACC
+#define USE_ACC
+#define USE_GYRO
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACCGYRO_BMI270
+#define GYRO_1_EXTI_PIN                 PC4
+#define ENSURE_MPU_DATA_READY_IS_LOW
+#define GYRO_1_CS_PIN                   SPI1_NSS_PIN
+#define GYRO_1_SPI_INSTANCE             SPI1
+#define GYRO_1_ALIGN                    CW0_DEG
+
+// OSD
+#define USE_MAX7456
+#define MAX7456_SPI_INSTANCE            SPI2
+#define MAX7456_SPI_CS_PIN              SPI2_NSS_PIN
+
+// FLASH
+#define USE_FLASH
+#define USE_FLASH_M25P16
+#define USE_FLASH_W25M
+#define USE_FLASH_W25M512
+#define FLASH_SPI_INSTANCE              SPI3
+#define FLASH_CS_PIN                    SPI3_NSS_PIN
+#define DEFAULT_BLACKBOX_DEVICE         BLACKBOX_DEVICE_FLASH
+
+// BARO & MAG
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define I2C1_SCL_PIN                    PB8
+#define I2C1_SDA_PIN                    PB9
+#define I2C2_SCL_PIN                    PH2
+#define I2C2_SDA_PIN                    PH3
+#define MAG_I2C_INSTANCE                I2CDEV_1
+#define BARO_I2C_INSTANCE               I2CDEV_1
+#define DEFAULT_BARO_DPS310
+
+// UART & VCP
+#define UART1_RX_PIN                    PA10
+#define UART1_TX_PIN                    PA9
+#define UART2_RX_PIN                    PA3
+#define UART2_TX_PIN                    PA2
+#define UART3_RX_PIN                    PC11
+#define UART3_TX_PIN                    PC10
+#define UART4_RX_PIN                    PA1
+#define UART4_TX_PIN                    PA0
+#define UART5_RX_PIN                    PD2
+#define UART5_TX_PIN                    PC12
+#define UART6_RX_PIN                    PC7
+#define UART6_TX_PIN                    PC6
+#define USB_DETECT_PIN                  PC5
+#define SERIALRX_UART                   SERIAL_PORT_USART2
+
+// ADC
+#define ADC_VBAT_PIN                    PC2
+#define ADC_CURR_PIN                    PC1
+#define ADC_INSTANCE                    ADC1
+#define ADC1_DMA_OPT                    11
+#define DEFAULT_CURRENT_METER_SOURCE    CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE    VOLTAGE_METER_ADC
+
+// PINIO
+#define PINIO1_PIN                      PC14
+
+// TIMERS & DMA
+#define TIMER_PIN_MAPPING               TIMER_PIN_MAP(  0, PA8,  1,  8 ) \
+                                        TIMER_PIN_MAP(  1, PA3,  1,  9 ) \
```

Comment:
RX_PPM does not need DMA, and this timer conflicts with motors 7 & 8.
```suggestion
                                        TIMER_PIN_MAP(  1, PA3,  3,  -1 ) \
```